### ISSUE TITLE
libadm 0.14.0 not via brew tap irt-open-source/homebrew-nga

### ISSUE THREAD

**Issue by WernerBleisteiner** [ebu/libadm#170]
Latest libadm 0.14.0 seems not to be brewable by 'brew tap irt-open-source/homebrew-nga'. 0.12.0 is the latest available.
So instruction in README is misleading.

**Comment by tomjnixon** [comment]
True. We've been trying to get that moved to the EBU organisation so that we  can update it (only ex-IRT people have access), but it's taking a while for some reason.

In the mean-time you can use https://github.com/rsjbailey/homebrew-nga ; see the instructions in this issue: https://github.com/ebu/libadm/issues/159

**Comment by WernerBleisteiner** [comment]
No issue with me. Just built it😉

**Comment by benjamin-weiss** [comment]
Why do you want to move the repository to the EBU org account? I would just create a fork in the EBU org account and keep the old irt repository as it is. Additionally you could add a message in the readme that this is now further maintained by the EBU. I have access to the irt repo and could merge a PR which adds such a message.

**Comment by tomjnixon** [comment]
Sorry i misspoke -- i meant making a copy/fork on the EBU account, rather than moving it.
